core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -330,7 +330,7 @@ export class BubbleCluster extends Chart {
     this._setUpFilters();
     const nodes = this.chart.selectAll(".bubble-group").data(
       root.descendants().filter(({ parent }) => !!parent),
-      (d) => d.data.category
+      (d) => d.data.title || d.data.label
```

Comment:
If it is parent bubbles that have children and a label instead of title

Reply:
Haha :D If this and that are exclusive `this || that` == `that || this`. If something has a title it is preferably used as a key. So sorry but I don't think this makes sense

Reply:
(And look at the parent nodes they are {label, children} so no title)

Reply:
yep yep makes sense was thinking about the data brought in from dataTypes rather than the data already assigned inside the nodes. I see what you mean about exclusivity 👍